src/ISA/ISA/Helper.fs

```diff
@@ -30,6 +30,13 @@ module ResizeArray =
             if f i then b.Add(i)
         b
 
+    let exists f (a : ResizeArray<_>) =
```

Comment:
This returns true or false and should be done using Seq.exists

Reply:
Ah oops